src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -262,7 +262,7 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 		let doNotRequestExtensionCompletions = false;
 		// Ensure that a key has been pressed since the last accepted completion in order to prevent
 		// completions being requested again right after accepting a completion
-		if (this._lastUserDataTimestamp < SuggestAddon.lastAcceptedCompletionTimestamp) {
+		if (this._promptInputModel.value !== '' && this._lastUserDataTimestamp < SuggestAddon.lastAcceptedCompletionTimestamp) {
```

Comment:
Not understanding how this would be hit since `_lastUserDataTimestamp` gets updated on `xterm.onKey`?

Reply:
there's no key entered in this case - just keybinding used - `ctrl+.`